fc/fractional_cover.py

```diff
@@ -87,7 +87,7 @@ def fractional_cover(nbar_tile: xarray.Dataset,
     error_val = -1
     where = numpy.where if not isinstance(output_data, dask_array_type) else dask.array.where
 
-    def data_func(measurement):
+    def data_func(measurement, shape=None):
```

Comment:
Given that shape is not used by the function, and that this seems to be purely for backwards compatibility with client modules - perhaps adding (unused) `*args, **kwargs` dynamic arguments be more Pythonic?